reference_data/platform_flavors.yaml

```diff
@@ -389,3 +389,42 @@
       url: "https://build.almalinux.org/pulp/content/copr/soksanichenko-Pungi-almalinux-$releasever-src-dr/"
       debug: false
       production: false
+
+- name: java-portable
+  data:
```

Comment:
Maybe the empty `data` field is causing the tests to fail?

Reply:
Fixed, but tests still failed

Reply:
No, that's not the issue. Looks like we were unable to construct coverage report:
"Error: TypeError: Cannot destructure property 'coverage' of 'report' as it is null.
Error: Cannot destructure property 'coverage' of 'report' as it is null."